atmos.yaml

```diff
@@ -91,6 +91,44 @@ logs:
   # Can also be set using 'ATMOS_LOGS_LEVEL' ENV var, or '--logs-level' command-line argument
   level: Info
 
+validate:
+  # The configuration settings for the editorconfig-checker.
+
+  editorconfig:
+    # A comma-separated list of file paths or patterns to exclude from checks.
+    exclude: ""
```

Comment:
This is wrong. It should be a list. It's also a list in their examples.

```
"Exclude": ["testfiles", "\\.md$"],
```

Reply:
Done